base/uk.ac.stfc.isis.ibex.ui.configserver/src/uk/ac/stfc/isis/ibex/ui/configserver/editing/blocks/BlocksEditorPanel.java

```diff
@@ -142,15 +163,59 @@ public void mouseDoubleClick(MouseEvent e) {
                 }
             }
         });
+        
+        /*
+         * Allows for a keyboard shortcut (Ctrl+c) to copy block.
+         */
```

Comment:
There is a question about consistency here. No other keyboard shortcuts exist in this view (for new block, delete block, etc). I think we should either add a complete set of keyboard shortcuts or none at all.